Projects.md

```diff
@@ -43,6 +43,10 @@ https://nicolasbarlogis.github.io/foxhole-logi-calculator/
 Standalone tool/website for extracting stockpile contents from screenshots  
 https://github.com/pogobanane/foxhole-screenparse
 
+#### Foxhole Inventory Report:
+Processes stockpile screenshots to produce graphical and spreadsheet reports.
+https://fir.gicode.net/
```

Comment:
Small nitpick, please make sure that this is on a newline in Markdown view. Markdown wont push line 48 to a new line unless line 47 ends with at least 2 spaces after the period.